### PR TITLE
centos5: bring playbook up to date, add monitoring

### PR DESCRIPTION
**Pull request by jbergstroem**
A bit of a mixed bag, but the most important thing here is that now use monit for monitoring.

### CODE REVIEW

File: setup/centos5/ansible-playbook.yaml [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
-    - include_vars: ansible-vars.yaml
-      tags: vars
-
-    - name: General | Bootstrap Python for Ansible
-      raw: yum -y install python-simplejson
-      tags: general
-
+    - name: Bootstrap | Install ansible requirements
+      yum: name=python-simplejson state=latest
+      when: python_version.stderr|search("2.4")
```
Comment (rvagg) [inline setup/centos5/ansible-playbook.yaml]: you've tested this?

**Comment by rvagg** [comment]
lgtm I think

File: setup/centos5/ansible-playbook.yaml (hunk 1 above)
Comment (jbergstroem) [inline setup/centos5/ansible-playbook.yaml]: Yes, spun up a new instance at DO a few hours ago.

Comment (jbergstroem) [inline setup/centos5/ansible-playbook.yaml]: Just fixed the yum invoke; sorry about that. Probably fiddled around after python was installed. Strange that ansible puts it in stderr when used while raw uses stdout. Oh well.

**Comment by jbergstroem** [comment]
Works properly now:

**python 2.4**
```
TASK [Bootstrap | Check python version] ****************************************
ok: [test-digitalocean-centos5-x64-3] => {"changed": false, "rc": 0, "stderr": "", "stdout": "Python 2.4.3\r\n", "stdout_lines": ["Python 2.4.3"]}

TASK [Bootstrap | Install ansible requirements] ********************************
ok: [test-digitalocean-centos5-x64-3] => {"changed": false, "rc": 0, "stderr": "", "stdout": "Loaded plugins: fastestmirror\r\nDetermining fastest mirrors\r\n * base: mirror.symnds.com\r\n * extras: mirrors.tripadvisor.com\r\n * updates: centos.mirror.constant.com\r\n\rbase <snip>
```

**python 2.6**
```
TASK [Bootstrap | Check python version] ****************************************
ok: [test-digitalocean-centos5-x64-3] => {"changed": false, "rc": 0, "stderr": "", "stdout": "Python 2.6.8\r\n", "stdout_lines": ["Python 2.6.8"]}

TASK [Bootstrap | Install ansible requirements] ********************************
skipping: [test-digitalocean-centos5-x64-3] => {"changed": false, "skip_reason": "Conditional check failed", "skipped": true}
```

**Pull request merged**